python/tvm/relay/frontend/mxnet_qnn_op_utils.py

```diff
@@ -277,6 +277,8 @@ def quantize_conv_weights_bias_channel_mkldnn_from_var(weights_var,
 def get_mkldnn_requantize_scale_outDtype(min_output_range,
                                          max_output_range,
                                          out_dtype):
+    """Get the MKLDNN requantized scale."""
+    quantized_out_range = (
```

Comment:
what's up with this line?

Reply:
These are a bunch of random edits that made the results of black happier.